integtest/Makefile

```diff
@@ -117,37 +120,155 @@ missing_include_fails_asciidoctor: missing_include.asciidoc
 /tmp/%_asciidoctor:
 	$(BD) --asciidoctor --doc $*.asciidoc
 
-.PHONY: small_all_expected_files
-small_all_expected_files: /tmp/small_all
-	[ -s $^/html/branches.yaml ]
-	grep '<a class="ulink" href="test/current/index.html" target="_top">Test book</a>' $^/html/index.html > /dev/null
-	grep '<meta http-equiv="refresh" content="0; url=current/index.html">' $^/html/test/index.html > /dev/null
-	[ -s $^/html/test/current/index.html ]
+.PHONY: simple_all_expected_files
+simple_all_expected_files:
+	# Test the simplest possible `--all` invocation
+	$(call BUILD_MINIMAL_ALL,/tmp/simple_all)
+	$(call MINIMAL_ALL_EXPECTED_FILES,/tmp/simple_all)
 
-.PRECIOUS: /tmp/small_all
-/tmp/small_all:
-	# Builds "--all" documentation specified by by the "small_conf.yaml" file.
+.PHONY: keep_hash_expected_files
+keep_hash_expected_files:
+	# Test that `--all --keep_hash` doesn't pull new updates
+	$(call BUILD_MINIMAL_ALL,/tmp/keep_hash)
 
-	# First build a repository to use as the source.
+	# REPLACE the minimal documentation in the source repo
+	cp ../README.asciidoc /tmp/source/index.asciidoc
+	cd /tmp/source && \
```

Comment:
Would the ever be run in parallel with itself? Might need a proper tmpdir.

Reply:
It'd be difficult to get it to run in parallel with itself but I may as well make a proper tmpdir. You are right about that!

Reply:
I guess Docker kinda takes care of that, doesn't it? I may have been thinking too old-school.